go-controller/pkg/ovn/libovsdbops/model_client_test.go

```diff
@@ -992,12 +998,19 @@ func TestCreateWithAdHocClient(t *testing.T) {
 	}
 
 	for _, tCase := range tt {
-		stopChan := make(chan struct{})
 		dbSetup := libovsdbtest.TestSetup{
 			SBData: tCase.initialDB,
 		}
 
-		nbClient, sbClient, _ := libovsdbtest.NewNBSBTestHarness(dbSetup, stopChan)
+		clientCleanup := libovsdbtest.NewCleanup()
```

Comment:
NIT: I seem to see these 2 always together now:

```
clientCleanup := libovsdbtest.NewCleanup()
serverCleanup := libovsdbtest.NewCleanup()
```

```
defer clientCleanup.Stop()
defer serverCleanup.Stop()
```
Assuming they always do go together, maybe we could create a helper function that does that, and then we save a line everywhere?

Something like:
```
clientCleanup, serverCleanup := libovsdbtest.NewCleanupPair()
defer libovsdbtest.StopCleanups(clientCleanup, serverCleanup)
```

Reply:
Yeah, I thought about that too. I'll do that, I can't think of a time when their order would be different.

Reply:
Done.